crates/ibc/src/core/ics04_channel/events/packet_attributes.rs

```diff
@@ -10,6 +10,7 @@ use crate::{
     timestamp::Timestamp,
 };
 use derive_more::From;
+use serde_derive::{Deserialize, Serialize};
```

Comment:
Can you remove all new `serde_derive` imports? This will cause conflicts with #331

Reply:
I added a comment at all the other ones so that we don't forget any